tests/Giraffe.Tests/Helpers.fs

```diff
@@ -196,9 +196,14 @@ let hasLastModified (lastModified: DateTimeOffset) (response: HttpResponseMessag
     Assert.Equal(lastModified, response.Content.Headers.LastModified.Value)
     response
 
+let getReqBody (ctx: HttpContext) =
```

Comment:
what is the difference with `getBody`?

Reply:
One reads from the Response, while the other reads from the Request

Reply:
Okay